README.md

```diff
@@ -99,7 +100,7 @@ The example below convert the video to mono audio.
 
 ```yaml
 env:
-  POSTPROCESSOR_ARGS: 'ffmpeg:-ac 1'
+  POSTPROCESSOR_ARGS: 'ExtractAudio+ffmpeg:-ac 1'
```

Comment:
Could you explain why this change is made?

Reply:
In newer `yt-dlp` versions the is an error in the old way.

The new way makes it explicit the phase to apply the command and fixes the error. 

This is obviously only an example of jow one would use the post processor commands.